examples/helm/alias.source

```diff
@@ -0,0 +1,35 @@
+function vhostport() {
+  vhost=$(minikube service $1 --url=true --format="{{.IP}}" | tail -n 1)
+  vport=$(minikube service $1 --url=true --format="{{.Port}}" | tail -n 1)
+
+  if [ -z $vport ]; then 
+    # This checks K8s running on an single node by kubeadm
+    if [ $(kubectl get nodes | grep -v NAM | wc -l) -eq 1 -o $(kubectl get nodes | grep -v NAM | grep master | wc -l ) -eq 1 ]; then
+      vhost="127.0.0.1"
+      vport=`kubectl describe service vtgate-zone1 | grep NodePort | grep mysql | awk '{print $3}' | awk -F'/' '{print $1}'`
+    fi
+  fi
+}
+
+function vsetmysql() {
+  vhostport vtgate-zone1
+  if [ -z $vport ]; then
+    echo "Error: failed to obtain [host:port] minikube or kubectl."
+    return 1
+  fi
+  echo "Setting alias mysql to: mysql -h $vhost -P $vport"
+  alias mysql="mysql -h $vhost -P $vport"
+}
+
+function vsetvclient() {
+  vhostport vtctld
+  if [ -z $vport ]; then
+    echo "Error: failed to: obtain [host:port] minikube or kubectl."
+    return 1
+  fi
+  echo "Setting alias vclient to vtctlclient -server=$vhost:$vport"
+  alias vclient="vtctlclient -server=$vhost:$vport"
```

Comment:
Can we change this to `vtctlclient` for now and then discuss new client? This will prevent creating a new "command" until we have a design.